ext/standard/basic_functions.c

```diff
@@ -2475,11 +2475,11 @@ ZEND_BEGIN_ARG_INFO_EX(arginfo_substr_compare, 0, 0, 3)
 	ZEND_ARG_INFO(0, case_sensitivity)
 ZEND_END_ARG_INFO()
 
-ZEND_BEGIN_ARG_INFO_EX(arginfo_utf8_encode, 0, 0, 1)
+ZEND_BEGIN_ARG_WITH_RETURN_TYPE_INFO_EX(arginfo_utf8_encode, 0, 1, IS_STRING, 0)
 	ZEND_ARG_INFO(0, data)
 ZEND_END_ARG_INFO()
 
-ZEND_BEGIN_ARG_INFO_EX(arginfo_utf8_decode, 0, 0, 1)
+ZEND_BEGIN_ARG_WITH_RETURN_TYPE_INFO_EX(arginfo_utf8_decode, 0, 1, IS_STRING, 0)
```

Comment:
Is there some case that utf8_(en|de)code return `false`?

https://github.com/php/php-src/blob/edd7fb2a16a17db017724831358a6706b4f69e40/ext/opcache/Optimizer/zend_func_info.c#L1050-L1051

Reply:
Nope, I just haven't gotten to updating those yet.

Reply:
Fixed in https://github.com/php/php-src/commit/673c70a4baf1f40ca043b05c6291252fe8701c50.